content/api/ip-pools.apib

```diff
@@ -146,20 +139,17 @@ Retrieves a specific IP Pool.
             }
 
 
-## Update an IP Pool [PUT]
+## Update an IP Pool [PUT /ip-pools/{id}]
 
-Updates an IP pool's name. To update sending IP pool assignments see [Sending IPs](/api/sending-ips/).
++ Data Structure
+    + name (string, required) - Name of the IP Pool
+    + signing_domain (string) - Domain to use as the DKIM verified signing domain
+    + fbl_signing_domain (string) - FBL Signing Domain for the pool. To unassign a custom FBL signing domain, provide an empty string for this field's value.
 
-To remove a signing domain, the value passed in should be the empty string.
 
-#### Request Body Attributes
-
-| Field          | Type           | Description                                         | Required      |
-|----------------|----------------|-----------------------------------------------------|---------------|
-| name           | string         | Name of the IP Pool                                 | yes           |
-| signing_domain | string         | Domain to use as the DKIM verified signing domain   | no            | 
-| fbl_signing_domain | string     | FBL Signing Domain for the pool | no       | To unassign a custom FBL signing domain, provide an empty string for this field's value |
+Updates an IP pool's name. To update sending IP pool assignments see [Sending IPs](/api/sending-ips/).
```

Comment:
Move above data structure and remove first sentence

Reply:
actually, this seems like more of an extra note, so could stay below, but still remove first sentence